client/src/pages/Faq.js

```diff
@@ -411,7 +411,7 @@ const Faq = () => {
           <details>
             <summary>How can I reset my password?</summary>
             <p>
-              Click on "Login" from the homepage and then click on "Forgot
+              Click on "Sign In" from the homepage and then click on "Forgot
               password" and follow the instructions
```

Comment:
Can you change line 389 and 424 to say something like `Your sign in credentials will be the same as` instead of your login credetials will be the same as..

Reply:
did you ask product about that copy?

Reply:
Yes, we want to have "sign in" instead of "login" in all places!! Thanks @stavares843

Reply:
Also, please replace all "Sign Up" to "Join Now"!!

Reply:
done